src/website/views/components/header.pug

```diff
@@ -803,7 +803,7 @@ ul#a-tabs--toolbar.a-tabs
   #a-tabs-panel--toolbar-base.a-tabs-panel.-active
     .-p--3(style="background-color:#eee;border-radius:.15em .15em 0 0;")
       header.o-header
-        nav.o-header__toolbar.-m--auto.-mw--980
```

Comment:
-m--auto has been correctly removed here as o-header flex configuration already provides centered position. But there are other -m--auto than can also be removed.